.travis.yml

```diff
@@ -0,0 +1,32 @@
+if: (type = push AND branch IN (master)) OR (type = pull_request)
+sudo: required
+services:
+  - docker
+
+env:
+  global:
+    - FLASK_APP=todo_app.app
+    - FLASK_ENV=production
+    - secure: TAtZ3ZDXZLhG7PeJBqutF6xXztmu1ZbZXc7ub3G7goSd5xQXcjWkPAyeOMBwIUEj4sRIUn/oWT/XVQejBjkHPO/KCrybd/yopHpVqZDFfGwD064v7EVZJisi4kcmfDzUGsAeZdNa2p1O1WUuhNVONzuqTEinarTS5GH5tS9S3quwdBKgTiyhzbV9eGIz9he6cEw5QbKlgL8hyJzRbTss9wxniTJ71TmT3iXXKg1HPZiQNIbqyn6iFZL7KSpS3ZlhkRed8XTInfuoufl747q3Nt2rKFqRG3JChBrd0FBOX0o0coTFhLTYsQnWGtrNh7zgWfu/Ad6onROeamlFXfIpiMKxybs/InUAUIH9/a/rjj5vkeO2MiCiH6d7Bm5QOKwtKWQbTSDq0D0JEqpmyNbawv9qe9cxRJcTeISSNAXiMCAyPn5MePj6Wyt/3V6tYPLw5/5lmrWhmkwgzdaSRRFvzatOekUNAyDZWbgzl+1eMIewTvskgQ947P7gQQEuRNR3bxrKlgwJKZHCDTZEj70W5J5VLA1l5bU1msddF/89/2hErcosiSNOnF9LhlmyhhEHOgOnn178Wb3wfrW0+cLK3RbqzHajbKCPTO+bfTdmj47T4ATwtbRkYlY0jk7fk0whA1BwHMnprYe259VcqvPnc2s8tiX0Zg/SuxkeW2fegjA=
+    - secure: jlQQBpU4Kx3KB3rzD8AUb1+2EggiLwWWchAeoxyAUJCgJ0jaQwegQJ8QGKek+g1zsUDTmPBP9zmIAF5CxZm4oqi08oB9U3yk9AtfL5s3eG0lFhOBNeVK3DyKQj2VB67sVmkXR78BGM+4pQoo44mnLlHvO0k1i0P9GArL0+aMGEJ4ut8aD6cCTozhvDmzicN2vgNJ/zQUOS8W1Rbl0sFZGuc8n/fsUC69bHzflLtsPAdHQyg5DW99uYvXq7voPinKICV4zugzHuqWGa7INz6bJAwGyYchL7pbCztppR30xOraj6yRotlovBekc/yKSJxumYZ+vy8aYV34pdC0XdRC0RE/gOJZ7kb1wDJKv4hECTQhlDVMws+OJ/WVUkBOBlVXoIqlrYGVV9guvIPOgLouCjX0foVjzXbpgiw+S5pZPs3dzD3dVibbGZ5KEC9JfiR1uQf+0qNko2jijgyqrRJ+jwvQ3NahckjckRVfx1Ww8x4QyCvKzkEPGPN2eaELxvIYwbZTDL41qFOCuGRopJkkT4LV8G7yGyls2eWwxFo7F3tlLqQNn8zPmZewhXwkoHeOyRoqSUT6FCOY1AzLcN56MJIZGz9YViAtcsOJfEJlBdvyYOPmBOvrBABVTY6atKSKROsKOGQgaui3qsTe658H9xAkpZmc26lsuaGhpnUJMT8=
+    - secure: U9GPJl+7EwQHRHaAb9e2/sIlwapBaDjxCAdK+3Of/ZDW0vhjlZ63dSUDBbHmx0SXRDlJUjwZQgqDyyO8biYne8jJWojeZmHYW5kxUeiq2Y87+GptGtSZ5r4YDDiTkr5bUjJhhby0GxKcEqo1ZraXOJTROmxkXTXC1UkW/9pUz6s1mEtmGJEEOU3gy4TGia0Mv0xZsoA2ZCNCVwFL9/9sTyQOGQlS1HD3QIA0ebfXPJlhoQMnMqzilo8cx+l8rY+yyBrztL9TMr1RZkNUtvnBy2Y+8XawnAZoe5rdfUNXb+3lUSnF6+p72qls/Z1Ab1S9e+MiIbpREnpfsYWS1Xepme+AWg0flKrOX4vi4Fi5B9l43nhEWgno9/5H42CJWWnc7zlSBsTKM/Nre8y3lzqFyoIMQUatquoQCVHBn/1JTBfA9lgjwONv8ZW8nnQbFcydvPl5dpgyPDODpU3wo3iKY8KiYTKiqJWX5pYzGsMxCuUEhfNm00pRc+14B7oRUgsOv9x2iuxd+rKV+rbCLF84q21GDYIwLeSudPNFiNVJg9a2IP91gdGXuWZnfRhv1dzXvUAvUBMZPOagL+SY2uXhEXq/SJyGDC1s5TMw8uHhRqD7dexzcPZxfNQuHaUnNWf1l6od/PREveuKcK9EaSObmifZiMbFmLZkVtlFtgOQXLM=
+    - COL_1="Not Started"
+    - COL_2="In Progress"
+    - COL_3="Completed"
+  COMPOSE_VERSION: 1.27.4
+
+before_install:
+  - curl -L https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
+script:
+  - echo "running tests for JB ToDo app"
+  - docker-compose --file docker-compose-travis.yml up --build
```

Comment:
You need to add `--exit-code-from todoapp` for this to actually have the right exit code (e.g. to fail the build if the todoapp service gets a failure running the tests)

Reply:
added this, thx